app/ascii_art.py

```diff
@@ -0,0 +1,201 @@
+menu_logo = r"""
+ ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄       ▄▄▄▄▄▄▄▄▄▄▄  ▄         ▄
+▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░▌     ▐░░░░░░░░░░░▌▐░▌       ▐░▌
+▐░█▀▀▀▀▀▀▀▀▀  ▀▀▀▀█░█▀▀▀▀ ▐░█▀▀▀▀▀▀▀▀▀ ▐░█▀▀▀▀▀▀▀█░▌▐░▌      ▀▀▀▀█░█▀▀▀▀ ▐░▌       ▐░▌
+▐░▌               ▐░▌     ▐░▌          ▐░▌       ▐░▌▐░▌          ▐░▌     ▐░▌       ▐░▌
+▐░█▄▄▄▄▄▄▄▄▄      ▐░▌     ▐░█▄▄▄▄▄▄▄▄▄ ▐░█▄▄▄▄▄▄▄█░▌▐░▌          ▐░▌     ▐░█▄▄▄▄▄▄▄█░▌
+▐░░░░░░░░░░░▌     ▐░▌     ▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░▌          ▐░▌     ▐░░░░░░░░░░░▌
+ ▀▀▀▀▀▀▀▀▀█░▌     ▐░▌     ▐░█▀▀▀▀▀▀▀▀▀ ▐░█▀▀▀▀▀▀▀█░▌▐░▌          ▐░▌     ▐░█▀▀▀▀▀▀▀█░▌
+          ▐░▌     ▐░▌     ▐░▌          ▐░▌       ▐░▌▐░▌          ▐░▌     ▐░▌       ▐░▌
+ ▄▄▄▄▄▄▄▄▄█░▌     ▐░▌     ▐░█▄▄▄▄▄▄▄▄▄ ▐░▌       ▐░▌▐░█▄▄▄▄▄▄▄▄▄ ▐░▌     ▐░▌       ▐░▌
+▐░░░░░░░░░░░▌     ▐░▌     ▐░░░░░░░░░░░▌▐░▌       ▐░▌▐░░░░░░░░░░░▌▐░▌     ▐░▌       ▐░▌
+ ▀▀▀▀▀▀▀▀▀▀▀       ▀       ▀▀▀▀▀▀▀▀▀▀▀  ▀         ▀  ▀▀▀▀▀▀▀▀▀▀▀  ▀       ▀         ▀
+
+           ▄▄▄▄▄▄▄▄▄▄▄  ▄         ▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄
+          ▐░░░░░░░░░░░▌▐░▌       ▐░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌
+          ▐░█▀▀▀▀▀▀▀▀▀ ▐░▌       ▐░▌▐░█▀▀▀▀▀▀▀▀▀ ▐░█▀▀▀▀▀▀▀▀▀ ▐░█▀▀▀▀▀▀▀▀▀
+          ▐░▌          ▐░▌       ▐░▌▐░▌          ▐░▌          ▐░▌
+          ▐░▌          ▐░█▄▄▄▄▄▄▄█░▌▐░█▄▄▄▄▄▄▄▄▄ ▐░█▄▄▄▄▄▄▄▄▄ ▐░█▄▄▄▄▄▄▄▄▄
+          ▐░▌          ▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌
+          ▐░▌          ▐░█▀▀▀▀▀▀▀█░▌▐░█▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀█░▌ ▀▀▀▀▀▀▀▀▀█░▌
+          ▐░▌          ▐░▌       ▐░▌▐░▌                    ▐░▌          ▐░▌
+          ▐░█▄▄▄▄▄▄▄▄▄ ▐░▌       ▐░▌▐░█▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄█░▌ ▄▄▄▄▄▄▄▄▄█░▌
+          ▐░░░░░░░░░░░▌▐░▌       ▐░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌
+           ▀▀▀▀▀▀▀▀▀▀▀  ▀         ▀  ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀
+"""
+
+FEN = r"rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1"
+KING = r"""
+   .::.
+   _::_
+ _/____\_
+ \      /
+  \____/
+  (____)
+   |  |
+   |__|
+  /    \
+ (______)
+(________)
+/________\
+"""
+QUEEN = r"""
+
+
+    ()
+  <~~~~>
+   \__/
+  (____)
+   |  |
+   |  |
+   |__|
+  /____\
+ (______)
+(________)
+"""
+BISHOP = r"""
+
+
+
+
+   <>_
+ (\)  )
+  \__/
+ (____)
+  |  |
+  |__|
+ /____\
+(______)
+"""
+ROOK = r"""
+
+
+
+
+
+
+ WWWWWW
+  |  |
+  |  |
+  |__|
+ /____\
+(______)
+"""
+PAWN = r"""
+
+
+
+
+
+
+
+  __
+ (  )
+  ||
+ /__\
+(____)
+"""
+KNIGHT = r"""
+
+
+
+
+
+ __/'''\
+]___ 0  }
+    /   }
+  /~    }
+  \____/
+  /____\
+ (______)
+"""
+KING_I = r"""
```

Comment:
Didn't @OculusMode made a numpy-magic code to invert this? We could just use that then?

Reply:
no we talked about this its fine this way